app/src/main/java/info/nightscout/android/medtronic/MainActivity.java

```diff
@@ -363,6 +382,9 @@ protected void onStart() {
     @Override
     protected void onPostCreate(Bundle savedInstanceState) {
         super.onPostCreate(savedInstanceState);
+//        sendStatus("Nightscout 600 Series Uploader");
```

Comment:
OK - that was WIP as unsure whether to notify user of poll intervals on startup. Will leave this kind of stuff out.